### PR TITLE
integrating with remarkjs

### PR DESCRIPTION
**Pull request by rnaresh**

### CODE REVIEW

**Comment by rnaresh** [comment]
https://github.com/wavemaker/docs/issues/133 - Took one of the usecase as mentioned in the ticket and fixed it for the dashboard.

Other useful resources: 
Markdown - https://github.com/gnab/remark/wiki/Markdown
Formatting - https://github.com/gnab/remark/wiki/Formatting
Custom Themes - https://github.com/gnab/remark/wiki/HTML-CSS-JS-samples

**Comment by rnaresh** [comment]
Because of the remark lib throwing Mixed Content
`
Mixed Content: The page at 'https://pr-174.dpwfbpxczhwpr.amplifyapp.com/learn/app-development/wavemaker-overview/product-walkthrough#dashboard-walkthrough' was loaded over HTTPS, but requested an insecure script 'http://remarkjs.com/downloads/remark-latest.min.js'. This request has been blocked; the content must be served over HTTPS.
`

Local testing is happening over http - So working fine in local

I'll check this issue

**Comment by rnaresh** [comment]
@prasanth-reddy-wavemaker-com this is fixed now. Check this URL - https://pr-174.dpwfbpxczhwpr.amplifyapp.com/learn/app-development/wavemaker-overview/product-walkthrough#dashboard-walkthrough

**Comment by prasanth-reddy-wavemaker-com** [comment]
Thanks @rnaresh. This is looking really good. One usability issues though:

- [ ] The embedded slideshow does not look like a slideshow. We should style the frame so it looks closer to how the google embedded slideshow looks. With a footer that has controls to go to next slide, goto full screen etc.

**Comment by prasanth-reddy-wavemaker-com** [comment]
Check out how remark was integrated into hugo - a similar static site generator on top of markdown files https://github.com/gnab/remark/wiki/Using-with-Hugo

**Comment by rnaresh** [comment]
### **All the possible wavemaker themes using remarkjs** 

remarkjs-wavemaker-theme-templates

### **Below screen shows after integrating one of themes to docs site**

learn-dashboard

@prasanth-reddy-wavemaker-com

File: website/slides/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+</div>
+
+---
+class:wave-theme,master
+
+### Dashboard
+Once you have created projects, the **Project Dashboard** page displays all your Projects, Prefabs and Template Bundles along with the latest activity updates.
+
+---
+layout:true
```
Comment (prasanth-reddy-wavemaker-com) [inline website/slides/dashboard.md:21]: Should this HTML code be repeated in every slideshow markdown file? Is there a way to reference it from one place, so there is no duplication?

File: website/slides/dashboard.md [hunk 2]
```diff
@@ -19,18 +8,8 @@ Once you have created projects, the **Project Dashboard** page displays all you
...
-        <img class="image logo" width="70"/>
-        <span>WaveMaker.Inc. All Rights Reserved</span>
-    </div>
-    <div class="action-btns col">
-        <button title="Previous (←)" class="previousSlide"><div class="arrow-left"></div></button>
-        <button title="Next (→)" class="nextSlide"><div class="arrow-right"></div></button>
-        <button title="Full screen (F)" class="fullScreen"><i class="wms wms-fullscreen"></i></button>
-    </div>
-</div>
+{{header}}
```
Comment (prasanth-reddy-wavemaker-com) [inline website/slides/dashboard.md:11]: Looks good.

**Pull request merged**